### ISSUE TITLE
Ajax test failure on Android 4.0-4.3

### ISSUE THREAD

**Issue by Krinkle** [jquery/jquery#3742]
Before:
* <http://swarm.jquery.org/job/5785>
* Commit 3fcddd6e72 (jQuery Periodic, 18 July 2017)

<img width="1201" alt="screen shot 2017-07-31 at 14 45 02" src="https://user-images.githubusercontent.com/156867/28799806-dce18c0c-75fe-11e7-80ed-707c578182d0.png">

After:
* <http://swarm.jquery.org/job/5848>
* Commit 262acc6f1e (jQuery Periodic, 24 July 2017)

<img width="1199" alt="screen shot 2017-07-31 at 14 45 25" src="https://user-images.githubusercontent.com/156867/28799822-f118893c-75fe-11e7-94dc-439597813815.png">

Possibly relevant commits:
* 928c580a1a  `Tests: Reduce the abort timeout for simple focus testing`
* deba37e53d `Tests: Simulate events when CI hinders use of native ones`
* 262acc6f1e `Ajax: add an ontimeout handler to all requests` (@timmywil, gh-3586, gh-3590)

**Comment by mgol** [comment]
Looks like a dupe of #3743? Why single out ajax failures when all tests just timeout?

**Comment by Krinkle** [comment]
@mgol This report an assertion failure (1 test failure) in the Ajax test suite, happening on Android 4 (Android 4.1, 4.2, 4.3 and presumably 4.0 as well). See the screenshot and difference between before/after. The time outs in all tests Android 4.0 are a separate issue introduced in the previous job, and presumably not Ajax-related.

**Comment by mgol** [comment]
Ah, it's about 4.x, not just 4.0, I misread.

**Comment by raphaellemes** [comment]
mano se pa ta zuado isso ae

**Comment by timmywil** [comment]
It seems Android 4.x does not call the ontimeout handler. This test was added when we added an ontimeout responder. The test ensures that we actually respond to native calls to `ontimeout`. If the browser never calls `ontimeout`, we aren't necessarily doing something wrong.

I see 2 options. (1) Add a workaround to use a setTimeout and call ontimeout ourselves manually, but that sounds error-prone and hacky. (2) Don't run this test in Android 4.x.

I lean towards 2.
